src/main/java/org/apache/sling/feature/analyser/task/AnalyzerTaskProvider.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.sling.feature.analyser.task;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.ServiceLoader;
+import java.util.Set;
+
+import org.apache.sling.feature.analyser.Analyser;
+
+public final class AnalyzerTaskProvider {
+
+    private AnalyzerTaskProvider() {
+        // this class must not be instantiated directly
+    }
+
+    public static AnalyserTask[] getTasks() {
```

Comment:
The methods don't need to be public either, I guess?

Reply:
Uhm I implemented the provider under the `.task` package (where the `AnalyserTask` is defined), that means methods would not be visible to the `Analyzer` class...

Reply:
Ah, sorry, missed that. 
Thanks for clarifying, @simonetripodi !

Reply:
My pleasure, always glad to cooperate! :)